docs/src/man/getting_started.md

```diff
@@ -355,7 +355,12 @@ we can observe that:
 
 #### Indexing syntax
 
-Specific subsets of a data frame can be extracted using the indexing syntax, similar to matrices. The colon `:` indicates that all items (rows or columns depending on its position) should be retained:
+Specific subsets of a data frame can be extracted using the indexing syntax,
+similar to matrices. In the [Indexing](@ref) section of the manual you can find
```

Comment:
@mortenpi - "Indexing" is a page name (it does not appear after `#` header in the file). Is this OK to make such a reference?

Reply:
Yep, this should work. I would build locally just in case to make sure it works, but it's probably fine -- no warnings in the logs.